lua/terrortown/menus/gamemode/gameplay/general.lua

```diff
@@ -34,6 +34,6 @@ function CLGAMEMODESUBMENU:Populate(parent)
     form2:MakeCheckBox({
         label = "label_crosshair_ironsight_low_enabled",
         convar = "ttt_ironsights_lowered",
-        master = crossEnb,
+        master = crossEnb, -- FIXME: does this even work? `crossEnb` is a checkbox local to the `appearance/crosshair` settings
```

Comment:
no, this doesn't work. This is an artifact from when this was in a different menu. But even then this should not have been tied to the state of the crosshair

Reply:
alright i'll remove this then

Reply:
done in [7ed4347](https://github.com/TTT-2/TTT2/pull/1574/commits/7ed4347fafa536cdf05d130afe3b4dabd48c199e)